content/blog/simplify-kubernetes-rbac-in-amazon-eks-with-open-source-pulumi-packages/index.md

```diff
@@ -10,9 +10,10 @@ One of the most common areas Kubernetes operators struggle with in
 production involves creating and managing role-based access control
 (RBAC). This is so daunting that RBAC is often not implemented, or
 implemented halfway, or the configuration becomes impossible to
-maintain. In this post, we will contrast the traditional way of working
-with RBAC on EKS with using Pulumi --- Pulumi makes RBAC on Kubernetes
-so easy that you'll never create an insecure cluster again!
+maintain. 
+
+Fortunately, Pulumi makes RBAC on Kuberenetes so easy that you'll never create an insecure cluster again. In this post, we will contrast the traditional way of working
+with RBAC on EKS with using Pulumi. Here are a few highlights:
```

Comment:
Should we move `Here are a few highlights:` below the `<!--more-->`?

Reply:
thanks you for checking! Fixed.